tests/e2e/query.go

```diff
@@ -74,20 +74,48 @@ func queryGaiaAllBalances(endpoint, addr string) (sdk.Coins, error) {
 	return balancesResp.Balances, nil
 }
 
-func queryGlobalFees(endpoint string) (amt sdk.DecCoins, err error) {
-	body, err := httpGet(fmt.Sprintf("%s/gaia/globalfee/v1beta1/minimum_gas_prices", endpoint))
+func queryGlobalFees(endpoint string) (sdk.DecCoins, error) {
+	body, err := httpGet(fmt.Sprintf("%s/gaia/globalfee/v1beta1/params", endpoint))
 	if err != nil {
 		return nil, fmt.Errorf("failed to execute HTTP request: %w", err)
 	}
 
-	var fees globalfee.QueryMinimumGasPricesResponse
+	var fees globalfee.QueryParamsResponse
 	if err := cdc.UnmarshalJSON(body, &fees); err != nil {
 		return sdk.DecCoins{}, err
 	}
 
 	return fees.MinimumGasPrices, nil
 }
 
+func queryBypassMsgs(endpoint string) ([]string, error) {
+	body, err := httpGet(fmt.Sprintf("%s/gaia/globalfee/v1beta1/params", endpoint))
```

Comment:
Didn't we have some func, that would do the error checking and unmarshaling to the supplied struct?

Reply:
Not that I'm aware of :\ Any idea @yaruwangway ?

Reply:
not aware of that